Compilation error while running cargo test

**Describe the bug**

After cloning the swc repository and following [the development environment setup guide](https://github.com/swc-project/swc/blob/master/CONTRIBUTING.md#getting-your-development-environment-set-up), I get a compilation error while running the cargo test command.

```console
$ cargo test --all --all-features
error[E0119]: conflicting implementations of trait `swc_common::EqIgnoreSpan` for type `decl::VarDeclKind`:
  --> ecmascript/ast/src/decl.rs:80:73
   |
80 | #[derive(StringEnum, Clone, Copy, Eq, PartialEq, PartialOrd, Ord, Hash, EqIgnoreSpan)]
   |                                                                         ^^^^^^^^^^^^ conflicting implementation for `decl::VarDeclKind`
81 | #[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary, EqIgnoreSpan))]
   |                                                                ------------ first implementation here
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0119`.
error: could not compile `swc_ecma_ast`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error[E0119]: conflicting implementations of trait `swc_common::EqIgnoreSpan` for type `decl::VarDeclKind`:
  --> ecmascript/ast/src/decl.rs:80:73
   |
80 | #[derive(StringEnum, Clone, Copy, Eq, PartialEq, PartialOrd, Ord, Hash, EqIgnoreSpan)]
   |                                                                         ^^^^^^^^^^^^ conflicting implementation for `decl::VarDeclKind`
81 | #[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary, EqIgnoreSpan))]
   |                                                                ------------ first implementation here
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0119`.
error: build failed
```

I'm not sure if this is an issue with the project, my environment, or something else (not terribly familiar with rust yet). Here's some more info about my environment:

```console
$ rustup show
Default host: x86_64-apple-darwin
rustup home:  /Users/danwoz/.rustup

installed toolchains
--------------------

stable-x86_64-apple-darwin
nightly-2020-11-24-x86_64-apple-darwin

active toolchain
----------------

nightly-2020-11-24-x86_64-apple-darwin (overridden by '/Users/danwoz/swc/rust-toolchain')
rustc 1.50.0-nightly (d9a105fdd 2020-11-23)

$ git rev-parse HEAD
0713f98e53db896f38d3476201cac182a33ad703
```

Let me know if there's any more information I can provide to help.

--- a/.github/workflows/compilation.yml
+++ b/.github/workflows/compilation.yml
@@ -38,6 +38,10 @@ jobs:
         run: |
           (cd common && cargo hack check --feature-powerset --no-dev-deps)
 
+      - name: Check swc_ecma_ast
+        run: |
+          (cd ecmascript/ast && cargo hack check --feature-powerset --no-dev-deps)
+
       - name: Check swc_ecma_transforms
         run: |
           (cd ecmascript/transforms && cargo hack check --feature-powerset --no-dev-deps)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.2.4"
+version = "0.2.5"
 
 [lib]
 name = "swc"
@@ -27,12 +27,12 @@ serde_json = "1"
 sourcemap = "6"
 swc_atoms = {version = "0.2", path = "./atoms"}
 swc_common = {version = "0.10", path = "./common", features = ["sourcemap", "concurrent"]}
-swc_ecma_ast = {version = "0.37.1", path = "./ecmascript/ast"}
-swc_ecma_codegen = {version = "0.43.3", path = "./ecmascript/codegen"}
-swc_ecma_ext_transforms = {version = "0.2.2", path = "./ecmascript/ext-transforms"}
-swc_ecma_parser = {version = "0.45.2", path = "./ecmascript/parser"}
-swc_ecma_preset_env = {version = "0.3.4", path = "./ecmascript/preset_env"}
-swc_ecma_transforms = {version = "0.33.4", path = "./ecmascript/transforms", features = [
+swc_ecma_ast = {version = "0.37.2", path = "./ecmascript/ast"}
+swc_ecma_codegen = {version = "0.43.4", path = "./ecmascript/codegen"}
+swc_ecma_ext_transforms = {version = "0.2.3", path = "./ecmascript/ext-transforms"}
+swc_ecma_parser = {version = "0.45.3", path = "./ecmascript/parser"}
+swc_ecma_preset_env = {version = "0.3.5", path = "./ecmascript/preset_env"}
+swc_ecma_transforms = {version = "0.33.5", path = "./ecmascript/transforms", features = [
   "compat",
   "module",
   "optimization",
@@ -40,8 +40,8 @@ swc_ecma_transforms = {version = "0.33.4", path = "./ecmascript/transforms", fea
   "react",
   "typescript",
 ]}
-swc_ecma_utils = {version = "0.27.1", path = "./ecmascript/utils"}
-swc_ecma_visit = {version = "0.23.1", path = "./ecmascript/visit"}
+swc_ecma_utils = {version = "0.27.2", path = "./ecmascript/utils"}
+swc_ecma_visit = {version = "0.23.2", path = "./ecmascript/visit"}
 swc_visit = {version = "0.2", path = "./visit"}
 
 [dev-dependencies]

--- a/bundler/Cargo.toml
+++ b/bundler/Cargo.toml
@@ -9,7 +9,7 @@ include = ["Cargo.toml", "build.rs", "src/**/*.rs", "src/**/*.js"]
 license = "Apache-2.0/MIT"
 name = "swc_bundler"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.20.4"
+version = "0.20.5"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
@@ -31,19 +31,19 @@ relative-path = "1.2"
 retain_mut = "0.1.2"
 swc_atoms = {version = "0.2.4", path = "../atoms"}
 swc_common = {version = "0.10.0", path = "../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ecmascript/ast"}
-swc_ecma_codegen = {version = "0.43.3", path = "../ecmascript/codegen"}
-swc_ecma_parser = {version = "0.45.2", path = "../ecmascript/parser"}
-swc_ecma_transforms = {version = "0.33.4", path = "../ecmascript/transforms", features = ["optimization"]}
-swc_ecma_utils = {version = "0.27.1", path = "../ecmascript/utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../ecmascript/visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ecmascript/ast"}
+swc_ecma_codegen = {version = "0.43.4", path = "../ecmascript/codegen"}
+swc_ecma_parser = {version = "0.45.3", path = "../ecmascript/parser"}
+swc_ecma_transforms = {version = "0.33.5", path = "../ecmascript/transforms", features = ["optimization"]}
+swc_ecma_utils = {version = "0.27.2", path = "../ecmascript/utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../ecmascript/visit"}
 
 [dev-dependencies]
 hex = "0.4"
 ntest = "0.7.2"
 reqwest = {version = "0.10.8", features = ["blocking"]}
 sha-1 = "0.9"
-swc_ecma_transforms = {version = "0.33.4", path = "../ecmascript/transforms", features = ["react", "typescript"]}
+swc_ecma_transforms = {version = "0.33.5", path = "../ecmascript/transforms", features = ["react", "typescript"]}
 tempfile = "3.1.0"
 testing = {version = "0.10.0", path = "../testing"}
 url = "2.1.1"

--- a/ecmascript/Cargo.toml
+++ b/ecmascript/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecmascript"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.18.4"
+version = "0.18.5"
 
 [features]
 codegen = ["swc_ecma_codegen"]
@@ -24,12 +24,12 @@ react = ["swc_ecma_transforms/react"]
 typescript = ["swc_ecma_transforms/typescript"]
 
 [dependencies]
-swc_ecma_ast = {version = "0.37.1", path = "./ast"}
-swc_ecma_codegen = {version = "0.43.3", path = "./codegen", optional = true}
-swc_ecma_dep_graph = {version = "0.12.2", path = "./dep-graph", optional = true}
-swc_ecma_parser = {version = "0.45.2", path = "./parser", optional = true}
-swc_ecma_transforms = {version = "0.33.4", path = "./transforms", optional = true}
-swc_ecma_utils = {version = "0.27.1", path = "./utils", optional = true}
-swc_ecma_visit = {version = "0.23.1", path = "./visit", optional = true}
+swc_ecma_ast = {version = "0.37.2", path = "./ast"}
+swc_ecma_codegen = {version = "0.43.4", path = "./codegen", optional = true}
+swc_ecma_dep_graph = {version = "0.12.3", path = "./dep-graph", optional = true}
+swc_ecma_parser = {version = "0.45.3", path = "./parser", optional = true}
+swc_ecma_transforms = {version = "0.33.5", path = "./transforms", optional = true}
+swc_ecma_utils = {version = "0.27.2", path = "./utils", optional = true}
+swc_ecma_visit = {version = "0.23.2", path = "./visit", optional = true}
 
 [dev-dependencies]

--- a/ecmascript/ast/Cargo.toml
+++ b/ecmascript/ast/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_ast"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.37.1"
+version = "0.37.2"
 
 [features]
 default = []

--- a/ecmascript/ast/src/decl.rs
+++ b/ecmascript/ast/src/decl.rs
@@ -78,7 +78,7 @@ pub struct VarDecl {
 }
 
 #[derive(StringEnum, Clone, Copy, Eq, PartialEq, PartialOrd, Ord, Hash, EqIgnoreSpan)]
-#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary, EqIgnoreSpan))]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
 pub enum VarDeclKind {
     /// `var`
     Var,

--- a/ecmascript/codegen/Cargo.toml
+++ b/ecmascript/codegen/Cargo.toml
@@ -7,7 +7,7 @@ include = ["Cargo.toml", "src/**/*.rs"]
 license = "Apache-2.0/MIT"
 name = "swc_ecma_codegen"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.43.3"
+version = "0.43.4"
 
 [dependencies]
 bitflags = "1"
@@ -15,9 +15,9 @@ num-bigint = {version = "0.2", features = ["serde"]}
 sourcemap = "6"
 swc_atoms = {version = "0.2", path = "../../atoms"}
 swc_common = {version = "0.10.0", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
 swc_ecma_codegen_macros = {version = "0.5", path = "./macros"}
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
 
 [dev-dependencies]
 swc_common = {version = "0.10.0", path = "../../common", features = ["sourcemap"]}

--- a/ecmascript/dep-graph/Cargo.toml
+++ b/ecmascript/dep-graph/Cargo.toml
@@ -6,14 +6,14 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_dep_graph"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.12.2"
+version = "0.12.3"
 
 [dependencies]
 swc_atoms = {version = "0.2", path = "../../atoms"}
 swc_common = {version = "0.10.1", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}
 
 [dev-dependencies]
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
 testing = {version = "0.10.0", path = "../../testing"}

--- a/ecmascript/ext-transforms/Cargo.toml
+++ b/ecmascript/ext-transforms/Cargo.toml
@@ -5,7 +5,7 @@ documentation = "https://swc.rs/rustdoc/swc_ecma_ext_transforms/"
 edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_ext_transforms"
-version = "0.2.2"
+version = "0.2.3"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
@@ -13,7 +13,7 @@ version = "0.2.2"
 phf = {version = "0.8.0", features = ["macros"]}
 swc_atoms = {version = "0.2", path = "../../atoms"}
 swc_common = {version = "0.10", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
-swc_ecma_utils = {version = "0.27.1", path = "../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
+swc_ecma_utils = {version = "0.27.2", path = "../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}

--- a/ecmascript/jsdoc/Cargo.toml
+++ b/ecmascript/jsdoc/Cargo.toml
@@ -5,7 +5,7 @@ documentation = "https://swc.rs/rustdoc/jsdoc/"
 edition = "2018"
 license = "Apache-2.0/MIT"
 name = "jsdoc"
-version = "0.13.2"
+version = "0.13.3"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
@@ -18,7 +18,7 @@ swc_common = {version = "0.10.0", path = "../../common"}
 [dev-dependencies]
 anyhow = "1"
 dashmap = "3"
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
 testing = {version = "0.10.0", path = "../../testing"}
 walkdir = "2"

--- a/ecmascript/parser/Cargo.toml
+++ b/ecmascript/parser/Cargo.toml
@@ -7,7 +7,7 @@ include = ["Cargo.toml", "src/**/*.rs", "examples/**/*.rs"]
 license = "Apache-2.0/MIT"
 name = "swc_ecma_parser"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.45.2"
+version = "0.45.3"
 
 [features]
 default = []
@@ -22,8 +22,8 @@ serde = {version = "1", features = ["derive"]}
 smallvec = "1"
 swc_atoms = {version = "0.2.3", path = "../../atoms"}
 swc_common = {version = "0.10.0", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}
 unicode-xid = "0.2"
 
 [dev-dependencies]

--- a/ecmascript/preset_env/Cargo.toml
+++ b/ecmascript/preset_env/Cargo.toml
@@ -5,7 +5,7 @@ documentation = "https://swc.rs/rustdoc/swc_ecma_preset_env/"
 edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_preset_env"
-version = "0.3.4"
+version = "0.3.5"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
@@ -20,14 +20,14 @@ st-map = "0.1.2"
 string_enum = {version = "0.3", path = "../../macros/string_enum"}
 swc_atoms = {version = "0.2", path = "../../atoms"}
 swc_common = {version = "0.10", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_transforms = {version = "0.33.4", path = "../transforms", features = ["compat", "proposal"]}
-swc_ecma_utils = {version = "0.27.1", path = "../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_transforms = {version = "0.33.5", path = "../transforms", features = ["compat", "proposal"]}
+swc_ecma_utils = {version = "0.27.2", path = "../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}
 walkdir = "2"
 
 [dev-dependencies]
 pretty_assertions = "0.6"
-swc_ecma_codegen = {version = "0.43.3", path = "../codegen"}
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
+swc_ecma_codegen = {version = "0.43.4", path = "../codegen"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
 testing = {version = "0.10", path = "../../testing"}

--- a/ecmascript/transforms/Cargo.toml
+++ b/ecmascript/transforms/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.33.4"
+version = "0.33.5"
 
 [features]
 compat = ["swc_ecma_transforms_compat"]
@@ -19,24 +19,24 @@ typescript = ["swc_ecma_transforms_typescript"]
 [dependencies]
 swc_atoms = {version = "0.2.0", path = "../../atoms"}
 swc_common = {version = "0.10.0", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "./base"}
-swc_ecma_transforms_compat = {version = "0.3.3", path = "./compat", optional = true}
-swc_ecma_transforms_module = {version = "0.3.3", path = "./module", optional = true}
-swc_ecma_transforms_optimization = {version = "0.3.4", path = "./optimization", optional = true}
-swc_ecma_transforms_proposal = {version = "0.3.3", path = "./proposal", optional = true}
-swc_ecma_transforms_react = {version = "0.3.3", path = "./react", optional = true}
-swc_ecma_transforms_typescript = {version = "0.3.4", path = "./typescript", optional = true}
-swc_ecma_utils = {version = "0.27.1", path = "../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "./base"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "./compat", optional = true}
+swc_ecma_transforms_module = {version = "0.3.4", path = "./module", optional = true}
+swc_ecma_transforms_optimization = {version = "0.3.5", path = "./optimization", optional = true}
+swc_ecma_transforms_proposal = {version = "0.3.4", path = "./proposal", optional = true}
+swc_ecma_transforms_react = {version = "0.3.4", path = "./react", optional = true}
+swc_ecma_transforms_typescript = {version = "0.3.5", path = "./typescript", optional = true}
+swc_ecma_utils = {version = "0.27.2", path = "../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}
 unicode-xid = "0.2"
 
 [dev-dependencies]
 pretty_assertions = "0.6"
 sourcemap = "6"
-swc_ecma_codegen = {version = "0.43.3", path = "../codegen"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "./testing"}
+swc_ecma_codegen = {version = "0.43.4", path = "../codegen"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "./testing"}
 tempfile = "3"
 testing = {version = "0.10.0", path = "../../testing"}
 walkdir = "2"

--- a/ecmascript/transforms/base/Cargo.toml
+++ b/ecmascript/transforms/base/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_base"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.2.3"
+version = "0.2.4"
 
 [dependencies]
 fxhash = "0.2.1"
@@ -16,11 +16,11 @@ scoped-tls = "1.0.0"
 smallvec = "1.6.0"
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_codegen = {version = "0.43.3", path = "../../codegen"}
+swc_ecma_codegen = {version = "0.43.4", path = "../../codegen"}
 testing = {version = "0.10", path = "../../../testing"}

--- a/ecmascript/transforms/compat/Cargo.toml
+++ b/ecmascript/transforms/compat/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_compat"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.3"
+version = "0.3.4"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
@@ -20,13 +20,13 @@ serde = {version = "1.0.118", features = ["derive"]}
 smallvec = "1.6.0"
 swc_atoms = {version = "0.2.5", path = "../../../atoms"}
 swc_common = {version = "0.10.8", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
 swc_ecma_transforms_macros = {version = "0.2.0", path = "../macros"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing"}
 testing = {version = "0.10.2", path = "../../../testing"}

--- a/ecmascript/transforms/module/Cargo.toml
+++ b/ecmascript/transforms/module/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_module"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.3"
+version = "0.3.4"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
@@ -16,13 +16,13 @@ indexmap = "1.6.1"
 serde = {version = "1.0.118", features = ["derive"]}
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_transforms_compat = {version = "0.3.3", path = "../compat"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing/"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "../compat"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing/"}
 testing = {version = "0.10", path = "../../../testing/"}

--- a/ecmascript/transforms/optimization/Cargo.toml
+++ b/ecmascript/transforms/optimization/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_optimization"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.4"
+version = "0.3.5"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
@@ -19,16 +19,16 @@ retain_mut = "0.1.2"
 serde_json = "1.0.61"
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_transforms_compat = {version = "0.3.3", path = "../compat"}
-swc_ecma_transforms_module = {version = "0.3.3", path = "../module"}
-swc_ecma_transforms_proposal = {version = "0.3.3", path = "../proposal"}
-swc_ecma_transforms_react = {version = "0.3.3", path = "../react"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing"}
-swc_ecma_transforms_typescript = {version = "0.3.4", path = "../typescript"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "../compat"}
+swc_ecma_transforms_module = {version = "0.3.4", path = "../module"}
+swc_ecma_transforms_proposal = {version = "0.3.4", path = "../proposal"}
+swc_ecma_transforms_react = {version = "0.3.4", path = "../react"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing"}
+swc_ecma_transforms_typescript = {version = "0.3.5", path = "../typescript"}

--- a/ecmascript/transforms/proposal/Cargo.toml
+++ b/ecmascript/transforms/proposal/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_proposal"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.3"
+version = "0.3.4"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
@@ -17,13 +17,13 @@ serde = {version = "1.0.118", features = ["derive"]}
 smallvec = "1.6.0"
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_transforms_compat = {version = "0.3.3", path = "../compat"}
-swc_ecma_transforms_module = {version = "0.3.3", path = "../module"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "../compat"}
+swc_ecma_transforms_module = {version = "0.3.4", path = "../module"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing"}

--- a/ecmascript/transforms/react/Cargo.toml
+++ b/ecmascript/transforms/react/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_react"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.3"
+version = "0.3.4"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
@@ -16,13 +16,13 @@ regex = "1.4.2"
 serde = {version = "1.0.118", features = ["derive"]}
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_transforms_compat = {version = "0.3.3", path = "../compat/"}
-swc_ecma_transforms_module = {version = "0.3.3", path = "../module"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing/"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "../compat/"}
+swc_ecma_transforms_module = {version = "0.3.4", path = "../module"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing/"}

--- a/ecmascript/transforms/testing/Cargo.toml
+++ b/ecmascript/transforms/testing/Cargo.toml
@@ -6,17 +6,17 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_testing"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.2.2"
+version = "0.2.3"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 swc_common = {version = "0.10", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_codegen = {version = "0.43.3", path = "../../codegen"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_codegen = {version = "0.43.4", path = "../../codegen"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 tempfile = "3.1.0"
 testing = {version = "0.10.2", path = "../../../testing"}

--- a/ecmascript/transforms/typescript/Cargo.toml
+++ b/ecmascript/transforms/typescript/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_transforms_typescript"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.3.4"
+version = "0.3.5"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
@@ -14,17 +14,17 @@ fxhash = "0.2.1"
 serde = {version = "1.0.118", features = ["derive"]}
 swc_atoms = {version = "0.2", path = "../../../atoms"}
 swc_common = {version = "0.10.9", path = "../../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../../ast"}
-swc_ecma_parser = {version = "0.45.2", path = "../../parser"}
-swc_ecma_transforms_base = {version = "0.2.3", path = "../base"}
-swc_ecma_utils = {version = "0.27.1", path = "../../utils"}
-swc_ecma_visit = {version = "0.23.1", path = "../../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../../ast"}
+swc_ecma_parser = {version = "0.45.3", path = "../../parser"}
+swc_ecma_transforms_base = {version = "0.2.4", path = "../base"}
+swc_ecma_utils = {version = "0.27.2", path = "../../utils"}
+swc_ecma_visit = {version = "0.23.2", path = "../../visit"}
 
 [dev-dependencies]
-swc_ecma_codegen = {version = "0.43.3", path = "../../codegen"}
-swc_ecma_transforms_compat = {version = "0.3.3", path = "../compat"}
-swc_ecma_transforms_module = {version = "0.3.3", path = "../module"}
-swc_ecma_transforms_proposal = {version = "0.3.3", path = "../proposal/"}
-swc_ecma_transforms_testing = {version = "0.2.2", path = "../testing"}
+swc_ecma_codegen = {version = "0.43.4", path = "../../codegen"}
+swc_ecma_transforms_compat = {version = "0.3.4", path = "../compat"}
+swc_ecma_transforms_module = {version = "0.3.4", path = "../module"}
+swc_ecma_transforms_proposal = {version = "0.3.4", path = "../proposal/"}
+swc_ecma_transforms_testing = {version = "0.2.3", path = "../testing"}
 testing = {version = "0.10.2", path = "../../../testing"}
 walkdir = "2.3.1"

--- a/ecmascript/utils/Cargo.toml
+++ b/ecmascript/utils/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_utils"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.27.1"
+version = "0.27.2"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
@@ -15,8 +15,8 @@ once_cell = "1"
 scoped-tls = "1"
 swc_atoms = {version = "0.2.0", path = "../../atoms"}
 swc_common = {version = "0.10.0", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
-swc_ecma_visit = {version = "0.23.1", path = "../visit"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
+swc_ecma_visit = {version = "0.23.2", path = "../visit"}
 unicode-xid = "0.2"
 
 [dev-dependencies]

--- a/ecmascript/visit/Cargo.toml
+++ b/ecmascript/visit/Cargo.toml
@@ -6,11 +6,11 @@ edition = "2018"
 license = "Apache-2.0/MIT"
 name = "swc_ecma_visit"
 repository = "https://github.com/swc-project/swc.git"
-version = "0.23.1"
+version = "0.23.2"
 
 [dependencies]
 num-bigint = {version = "0.2", features = ["serde"]}
 swc_atoms = {version = "0.2", path = "../../atoms"}
 swc_common = {version = "0.10.0", path = "../../common"}
-swc_ecma_ast = {version = "0.37.1", path = "../ast"}
+swc_ecma_ast = {version = "0.37.2", path = "../ast"}
 swc_visit = {version = "0.2.1", path = "../../visit"}

